element-hq/element-x-android
# Always display 'lost recovery key?' option

## Type of change


## Content

- Removes 'create new key' option from the initial screen of the session verification flow that appeared when the current session was the only one.
- Always displays 'Lost your recovery key?' action in the enter recovery key screen instead.
- Added tests for that screen.

## Motivation and context

Should be enough to close #2740. It also applies the latest changes to the designs [in Figma](https://www.figma.com/file/qjs3RKuofeKzBRPnOmLo2a/OIDC-sign-in-with-QR-code?type=design&node-id=36-60942&mode=design&t=zM2XwBeEOrau88Pu-4), which changed the behaviour mentioned above.

## Screenshots / GIFs

In the PR.

## Tests



- Log into an account.
- Use 'enter recovery key' to enter the recovery key verification screen.
- The new 'Lost your recovery key?' button should be present.

## Tested devices

- OS version(s): 14

## Checklist
